proto/redpanda/api/dataplane/v1alpha2/cloud_storage.proto

```diff
@@ -0,0 +1,394 @@
+syntax = "proto3";
+
+package redpanda.api.dataplane.v1alpha2;
+
+import "buf/validate/validate.proto";
+import "google/api/annotations.proto";
+import "google/api/field_behavior.proto";
+import "protoc-gen-openapiv2/options/annotations.proto";
+
+message MountTopicsRequest {
+  // TopicMount defines the migration of a topic from the cloud storage into this cluster,
+  // so that it becomes available via the Kafka API.
+  message TopicMount {
+    // SourceTopic is the topic name or full reference we want to mount. The full reference
+    // must be used in case the same topic exists more than once. This may be the case if
+    // the same topic has been unmounted multiple times. List all mountable topics to
+    // find the full reference (contains topic name, cluster uuid and revision).
+    string source_topic_reference = 1 [
+      (google.api.field_behavior) = REQUIRED,
+      (buf.validate.field).required = true,
+      (buf.validate.field).string.min_len = 1
+    ];
+    // Alias may be provided to mount the topic under a different alias. Leave
+    // blank to re-use the source topic name.
+    string alias = 2 [
+      (buf.validate.field).string.max_len = 249,
+      (buf.validate.field).string.pattern = "^[a-zA-Z0-9._\\-]*$"
+    ];
+  }
+
+  repeated TopicMount topics = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (buf.validate.field).required = true,
+    (buf.validate.field).repeated = {
+      min_items: 1
+      max_items: 1024
+    }
+  ];
+}
+
+message MountTopicsResponse {
+  int32 mount_task_id = 1;
```

Comment:
```suggestion
  // ID of mount 
  int32 mount_task_id = 1;
```
I noticed that it's the same field as with unmount, so adding a description just to more easily differentiate